build-win.ps1

```diff
@@ -5,11 +5,10 @@ param (
   [string]$enableWin10 = "true",
   [string]$enableMini = "true",
   [string]$enableTests = "true",
-  [string]$customProps = ""
+  [string]$customProps = "",
+  [string]$vsDevCmdBat = "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat"
```

Comment:
If possible, use the `%ProgramFilesx86%` variable.

Reply:
It is not easy because in build environment it could be empty, so logic could be complicated - not good for default parameter